stdlib/2and3/wsgiref/handlers.pyi

```diff
@@ -0,0 +1,88 @@
+import sys
+from types import TracebackType
+from typing import Optional, Dict, MutableMapping, Type, Text, Callable, List, Tuple, IO
+
+from .headers import Headers
+from .types import WSGIApplication, WSGIEnvironment, StartResponse, InputStream, ErrorStream
+from .util import FileWrapper, guess_scheme
+
+_exc_info = Tuple[Optional[Type[BaseException]],
+                  Optional[BaseException],
+                  Optional[TracebackType]]
+
+def format_date_time(timestamp: Optional[float]) -> str: ...  # undocumented
+if sys.version_info >= (3, 2):
+    def read_environ() -> Dict[str, str]: ...
+
+class BaseHandler:
+    wsgi_version: Tuple[int, int]  # undocumented
+    wsgi_multithread: bool
+    wsgi_multiprocess: bool
+    wsgi_run_once: bool
+
+    origin_server: bool
+    http_version: str
+    server_software: Optional[str]
+
+    os_environ: MutableMapping[str, str]
+
+    wsgi_file_wrapper: Optional[Type[FileWrapper]]
+    headers_class: Type[Headers]  # undocumented
+
+    traceback_limit: Optional[int]
+    error_status: str
+    error_headers: List[Tuple[Text, Text]]
+    error_body: bytes
+
+    def run(self, application: WSGIApplication) -> None: ...
+
+    def setup_environ(self) -> None: ...
+    def finish_response(self) -> None: ...
+    def get_scheme(self) -> str: ...
+    def set_content_length(self) -> None: ...
+    def cleanup_headers(self) -> None: ...
+    def start_response(self, status: Text, headers: List[Tuple[Text, Text]], exc_info: Optional[_exc_info] = ...) -> Callable[[bytes], None]: ...
+    def send_preamble(self) -> None: ...
+    def write(self, data: bytes) -> None: ...
+    def sendfile(self) -> bool: ...
+    def finish_content(self) -> None: ...
+    def close(self) -> None: ...
+    def send_headers(self) -> None: ...
+    def result_is_file(self) -> bool: ...
+    def client_is_modern(self) -> bool: ...
+    def log_exception(self, exc_info: _exc_info) -> None: ...
+    def handle_error(self) -> None: ...
+    def error_output(self, environ: WSGIEnvironment, start_response: StartResponse) -> List[bytes]: ...
+
+    def _write(self, data: bytes) -> None:
+        raise NotImplementedError()
```

Comment:
I don't think this does anything in stubs. Maybe we should decorate these with `@abc.abstractmethod` instead.